wgpu/Cargo.toml

```
@@ -131,7 +130,7 @@ log = "0.4"
 noise = { version = "0.7", default-features = false }
 obj = "0.10"
 png = "0.17"
-rand = "0.7.2"
+nanorand = "0.7"
```

```suggestion
nanorand = { version = "0.7", default-features = false, features = ["wyrand"] }
```

Just because people get worried about dependency size.